tools/nni_cmd/launcher.py

```diff
@@ -272,6 +273,17 @@ def set_experiment(experiment_config, mode, port, config_file_name):
 def launch_experiment(args, experiment_config, mode, config_file_name, experiment_id=None):
     '''follow steps to start rest server and start experiment'''
     nni_config = Config(config_file_name)
+
+    # check packages for tuner
+    if experiment_config.get('tuner') and experiment_config['tuner'].get('builtinTunerName'):
+        tuner_name = experiment_config['tuner']['builtinTunerName']
+        module_name = ModuleName[tuner_name]
+        try:
+            check_call([sys.executable, '-c', 'import %s'%(module_name)])
+        except:
```

Comment:
Shall we only catch Exception like "ImportError" or "ModuleNotFoundError" here? :)

Reply:
Sure, fixed.